src/production.js

```diff
@@ -17,11 +17,14 @@ export default new Config().merge({
       compress: {
         screw_ie8: true,
         warnings: false,
+        comparisons: false,
+      },
+      output: {
+        comments: false,
+        ascii_only: true,
       },
     }),
-    new webpack.optimize.AggressiveMergingPlugin(),
-    new webpack.optimize.MinChunkSizePlugin({
-      minChunkSize: 2048,
```

Comment:
Are these outdated for webpack 3? Or are `DefinePlugin` and `optimize.ModuleConcatenationPlugin` just better options?

Reply:
@calthomson not outdated, but there is some discussion around their usage x dynamic imports. So since we will be going with dynamic imports I removed them from the base. They still can be used in a config inside the project. The Define, is to make sure that webpack is running in production mode and ModuleConcatenationPlugin is to have hoisting and its exclusive for webpack 3

Reply:
Cool! Thanks for the explanation